packages/cli/src/commands/i18n/i18nScan.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import fs from 'fs';
+import glob from 'glob';
+import i18n from 'i18next-scanner';
+import { OptionValues } from 'commander';
+
+export default async (opts: OptionValues) => {
+  const parser = new i18n.Parser({
+    nsSeparator: false,
+    keySeparator: false,
+  });
+  const filePaths = glob.sync(opts.src, {
+    ignore: ['./node_modules/**', './**/*.test.tsx'],
```

Comment:
```suggestion
    ignore: ['**/node_modules/**', '**/*.test.tsx'],
```

does this work? not entirely sure what `.` means otherwise in this context